execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -1168,4 +1169,27 @@ where
         self.transforms = Vec::new();
         self
     }
+
+    /// Advances eras by num_eras
+    pub fn advance_eras_by(&mut self, num_eras: u64, mut reward_items: Vec<RewardItem>) {
```

Comment:
Why does `reward_items` has to be mutable?

Reply:
The Vec has ownership of the RewardItems which does not implement the copy trait. Thus, we need to pop off the rewards items one by one from reward_items Vec (which then must be declared mutable) and hand over ownership of the popped reward_item to the step_request_builder.